### PR TITLE
added container component

### PR DESCRIPTION
**Pull request by leannachan**
What I did:
added a container folder under cse-ui-kit.
created files: container-Styled.tsx, container.stories.tsx, container.tsx, index.ts.
created component Container look like below:
image

### CODE REVIEW

**Comment by james-teng-0** [comment]
The container looks good to me but I think you can remove the controls/button related stuff in the container.stories.tsx file so storyboard doesn't render it, since it doesn't do anything for the container.

image

**Review by james-teng-0 — DISMISSED** [review]
The container looks good to me but I think you can remove the controls/button related stuff in the container.stories.tsx file so storyboard doesn't render it, since it doesn't do anything for the container.

image

File: Frontend/src/cse-ui-kit/container/container.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+export default function Container({ className}: Props) {
+  return (
+    <div className={className}>
+      <header>
+          <StyledContainer></StyledContainer>
+      </header>      
+    </div>
+  );
+}
```
Comment (fafnirZ) [inline Frontend/src/cse-ui-kit/container/container.tsx:17]: @leannachan Container should take in a Children props, and should be 
```js
<StyledContainer>
  {children}
</StyledContainer>
```
this is so we can actually pass children into the container.

**Review by fafnirZ — CHANGES_REQUESTED** [review]
Overall good progress !!! Please address the changes and re-request review :)

File: Frontend/src/cse-ui-kit/container/container.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { StyledContainer, containerProps } from './container-Styled';
+
+
+type Props = {
+  className?: string;
+} & containerProps;
+
+export default function Container({ className}: Props) {
```
[the remark this answers is not in the document]
Comment (leannachan) [reply Frontend/src/cse-ui-kit/container/container.tsx]: Some changes after the reviews:
- removed the background colour for dotted container
- renamed the container to DottedContainer for all the functions, files and folders
- dotted container take in children props and removed div and header tags

**Comment by fafnirZ** [comment]
LGTM!!! Great work!! :), 1 more small change though, get rid of all the comments in storybook that links you to a guide, and after you're done with that feel free to merge it into main

**Review by fafnirZ — APPROVED** [review]

**Pull request merged**